src/App.tsx

```diff
@@ -30,9 +54,20 @@ function App() {
 
     return (
         <>
-            <Header activePage={activePage} handleActivePage={handleActivePage} totalProductsInCart={totalProductsInCart} />
-            {activePage === 'about' ? <AboutMe /> : <ProductList updateTotalCart={updateTotalCart} />}
-            <Footer />
+            <div className={`${currentTheme === 'dark' ? styles.dark : styles.light}`}>
```

Comment:
You need to test it, but I hope that when you use `document.html.className = theme;`  , you don't need this `<div>` there